### ISSUE TITLE
Documentation update for pingdom webhook

### ISSUE THREAD

**Issue by lotooo** [alerta/alerta-docs#98]
In order to make the pingdom webhooks work, I followed this https://docs.alerta.io/en/latest/integrations.html#id7

The example webhook URL is 

```
Example Pingdom Webhook URL

https://alerta.example.com/api/webhooks/pingdom
```

But in my case, I had to include the api_key to make it work https://alerta.example.com/api/webhooks/pingdom?api-key=xxxxxxx

I think the doc needs more details, either adding that info here or explaining how we can make the auth work with the URL provided.

I can PR any doc repo to update.

**Comment by satterly** [comment]
That's only an issue if you have enabled and are enforcing authentication. And in that case it's true for all webhooks, not jus for Pingdom. There is a specific web page that discusses how to configure authentication for webhooks and other HTTP requests. See https://docs.alerta.io/en/latest/authentication.html#api-keys
However, if you have some ideas of how to make this info more obvious or accessible then please go ahead and create a PR. Thanks!

**Comment by lotooo** [comment]
ohhh ok it was not clear to me (but yes I need authentication on others webhooks we are using but most of them have a configuration fiel where I could set the token.

Could we add something like this at the top of the webhook section [here](https://docs.alerta.io/en/latest/integrations.html#webhooks) to make it clearer ?

```
If authentication is enforced, then an API key is needed to access the alerta API programatically and use the webhooks. Please follow this page for more information on how to pass  your api-key : https://docs.alerta.io/en/latest/authentication.html#api-keys
```

Another way would be to specify it on each webhook section or give a different exemple URL with or without authentication enfocred ([like in the grafana webhook doc](https://docs.alerta.io/en/latest/integrations.html#grafana))

**Comment by satterly** [comment]
Either way is fine with me. 👍

**Comment by lotooo** [comment]
I didn't notice at first taht you moved my GHI to a different repo :man_facepalming: 

I created this PR : https://github.com/alerta/alerta-docs/pull/99

**Comment by satterly** [comment]
Fixed by #99
